package.json

```
@@ -67,6 +68,10 @@
     "test:db:setup": "DATABASE_URL=postgresql://postgres@localhost:5433/techtrend_test npx prisma migrate deploy && DATABASE_URL=postgresql://postgres@localhost:5433/techtrend_test npx tsx prisma/seed-test.ts",
     "test:db:reset": "DATABASE_URL=postgresql://postgres@localhost:5433/techtrend_test npx prisma migrate reset --force",
     "test:e2e": "playwright test",
+    "test:e2e:report": "npm run test:e2e 2>&1 | node scripts/e2e-reporter.js",
+    "test:e2e:smoke": "playwright test --grep @smoke",
+    "test:e2e:critical": "playwright test --grep @critical",
+    "test:e2e:full": "playwright test",
```

Biome (2.1.2)

[error] The key test:e2e:full was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)